src/main/java/org/sciborgs1155/robot/drive/Drive.java

```diff
@@ -1,263 +1,181 @@
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.jni.CANSparkMaxJNI;
+
 import edu.wpi.first.math.MathUtil;
-import edu.wpi.first.math.estimator.SwerveDrivePoseEstimator;
-import edu.wpi.first.math.filter.SlewRateLimiter;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.estimator.DifferentialDrivePoseEstimator;
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Rotation2d;
-import edu.wpi.first.math.geometry.Transform2d;
 import edu.wpi.first.math.kinematics.ChassisSpeeds;
-import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
-import edu.wpi.first.math.kinematics.SwerveModulePosition;
-import edu.wpi.first.math.kinematics.SwerveModuleState;
-import edu.wpi.first.math.util.Units;
-import edu.wpi.first.wpilibj.smartdashboard.Field2d;
-import edu.wpi.first.wpilibj.smartdashboard.FieldObject2d;
-import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import io.github.oblarg.oblog.Loggable;
 import io.github.oblarg.oblog.annotations.Log;
+
 import java.util.List;
-import java.util.function.DoubleSupplier;
-import org.photonvision.EstimatedRobotPose;
+import java.util.function.Supplier;
+
 import org.sciborgs1155.lib.failure.Fallible;
-import org.sciborgs1155.lib.failure.FaultBuilder;
 import org.sciborgs1155.lib.failure.HardwareFault;
-import org.sciborgs1155.robot.Constants;
-import org.sciborgs1155.robot.Robot;
+import org.sciborgs1155.robot.Ports.DrivePorts;
 
-public class Drive extends SubsystemBase implements Fallible, Loggable, AutoCloseable {
+import edu.wpi.first.math.kinematics.DifferentialDriveWheelSpeeds;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
 
-  @Log private final ModuleIO frontLeft;
-  @Log private final ModuleIO frontRight;
-  @Log private final ModuleIO rearLeft;
-  @Log private final ModuleIO rearRight;
 
-  private final List<ModuleIO> modules;
+public class Drive extends SubsystemBase implements Loggable, AutoCloseable{
+  //creates StandardDriveMotor
+  StandardDriveMotor driveMotor = new StandardDriveMotor();
+//new formatting, check out StandardDriveMotor for more; just added so it is less messy in this document
+  private final CANSparkMax FRmotor = driveMotor.create(DrivePorts.FR_DRIVE_PORT);
+  private final CANSparkMax FLmotor = driveMotor.create(DrivePorts.FL_DRIVE_PORT);
+  private final CANSparkMax MRmotor = driveMotor.create(DrivePorts.MR_DRIVE_PORT);
+  private final CANSparkMax MLmotor = driveMotor.create(DrivePorts.ML_DRIVE_PORT);
+  private final CANSparkMax BRmotor = driveMotor.create(DrivePorts.BL_DRIVE_PORT);
+  private final CANSparkMax BLmotor = driveMotor.create(DrivePorts.BL_DRIVE_PORT);
```

Comment:
More style nitpicks: variable names should be `camelCase`, so consider naming the motors `frontRight` `backLeft` etc